src/modules/selectors/blend.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 The Noise-rs Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use num_traits::Float;
+use math::interp;
+use NoiseModule;
+
+/// Noise module that outputs the weighted blend of the output values from two
+/// source modules given the output value supplied by a control module.
+#[derive(Clone, Debug)]
+pub struct Blend<Source1, Source2, Control> {
+    /// Outputs one of the values to blend.
+    pub source1: Source1,
+
+    /// Outputs one of the values to blend.
+    pub source2: Source2,
+
+    /// Determines the weight of the blending operation. Negative values weight
+    /// the blend towards the output value from the _Source1_ module. Positive
```

Comment:
Should we be using inline code (backtick delimiters) instead of italics when referring to the type parameters?